### PR TITLE
#20 : Mongodb configuration

### PR DESCRIPTION
**Pull request by jbhushan791**
MongoDb configuration

### CODE REVIEW

**Comment by hrishikeshpaul** [comment]
Can you add a README.md here - https://github.com/airavata-courses/scrapbook/tree/feature/jb/image-service/scrapbook-image-service - and write steps as to how we can install and run it, and if any dependencies are required!

File: scrapbook-image-service/src/main/java/com/iu/scrapbook/document/Image.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ */
+@Document
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Image {
+
+    @Id
+    private Long id;
```
Comment (chirag-indi) [inline scrapbook-image-service/src/main/java/com/iu/scrapbook/document/Image.java:21]: Isn't it better to use UUID instead of Long?

**Review by chirag-indi — DISMISSED** [review]

**Comment by jbhushan791** [comment]
> Can you add a README.md here - https://github.com/airavata-courses/scrapbook/tree/feature/jb/image-service/scrapbook-image-service - and write steps as to how we can install and run it, and if any dependencies are required!

File: scrapbook-image-service/src/main/java/com/iu/scrapbook/document/Image.java (hunk 1 above)
Comment (jbhushan791) [reply scrapbook-image-service/src/main/java/com/iu/scrapbook/document/Image.java:21]: Anything works as long as they all are unique. UUID is difficult to read so we generally avoids using it. But here we are not going to sho id to user so we can use UUID. I will change that.

**Comment by jbhushan791** [comment]
> Can you add a README.md here - https://github.com/airavata-courses/scrapbook/tree/feature/jb/image-service/scrapbook-image-service - and write steps as to how we can install and run it, and if any dependencies are required!

 Done this

**Comment by jbhushan791** [comment]
merging image service to deveop < mongo config>

**Pull request merged**